smurfs/src/components/Smurfs.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchSmurfs } from '../actions'
+
+class Smurfs extends React.Component {
+    constructor() {
```

Comment:
if you are not using the state there is no need to call the constructor or super.